src/TypeSpec.Extension/Emitter.Csharp/src/lib/clientModelBuilder.ts

```diff
@@ -221,6 +223,30 @@ export function createModelForService(
     setUsage(usages, modelMap);
     setUsage(usages, enumMap);
 
+    // propagate the isConfident from the types to the operations
+    for (const client of clients) {
+        for (const op of client.Operations) {
+            let isOperationConfident = true;
+            for (const parameter of op.Parameters) {
+                // skip the special parameters
+                if (
+                    parameter.IsApiVersion ||
+                    parameter.IsEndpoint ||
+                    parameter.IsContentType
+                ) {
+                    parameter.Type.IsConfident = true; // we never recognize these as low confident level
+                    continue;
+                }
+                isOperationConfident &&= isConfident(parameter.Type);
+            }
+            for (const response of op.Responses) {
+                isOperationConfident &&= isConfident(response.BodyType);
```

Comment:
If the response.BodyType is the base type and one of its derived type is lowConfidence, will this operation also be considered as low confidence?  Same for the request.

Reply:
Yes. The calculation part of IsConfident answers your question, code here is USING the calculated values.

This is how we calculate isConfident for a model:

1. check if it has baseModel. Set baseModel?.IsConfident as the intial value of `IsConfident` of this model.
2. go through all the properties to get the IsConfident value of this model.

Therefore if base is false, the initial value is false, and the result will be false as well.